src/__tests__/validateFieldsNatively.ts

```diff
@@ -0,0 +1,42 @@
+import { Field, FieldError, InternalFieldName } from 'react-hook-form';
+import { validateFieldsNatively } from '../validateFieldsNatively';
+
+const flatObject: Record<string, FieldError> = {
+  name: { type: 'st', message: 'first message' },
+};
+
+const fields = {
+  name: {
+    ref: {
+      reportValidity: jest.fn(),
+      setCustomValidity: jest.fn(),
+    },
+  },
+  nd: {
```

Comment:
minor stuff: `nd` means? 😆

Reply:
It's just for testing purpose, I'll improve it :)

Reply:
hahah don't worry about it.